packit_service/models.py

```diff
@@ -365,6 +365,9 @@ class ProjectAndEventsConnector:
     def get_project_event_model(self) -> Optional["ProjectEventModel"]:
         return self.runs[0].project_event if self.runs else None
 
+    def get_package_name(self) -> Optional[str]:
+        return self.runs[0].package_name
```

Comment:
I would just change this as the previous method is to be safe
```suggestion
        return self.runs[0].package_name if self.runs else None
```